packages/stdlib-shims/stdlib-shims.0.1.0/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+maintainer: "The stdlib-shims programmers"
+authors: "The stdlib-shims programmers"
+homepage: "http://ocaml.org"
+doc: "https://ocaml.github.io/stdlib-shims/"
+dev-repo: "git+https://github.com/ocaml/stdlib-shims.git"
+bug-reports: "https://github.com/ocaml/stdlib-shims/issues"
+tags: ["stdlib" "compatibility" "org:ocaml"]
+license: ["typeof OCaml system"]
+depends: [
+  "dune"
```

Comment:
```suggestion
  "dune" {build}
```
This most likely needs the `build` attribute

Reply:
Indeed